src/PhpSpreadsheet/Worksheet/Worksheet.php

```diff
@@ -2007,6 +2017,56 @@ public function removeAutoFilter(): self
         return $this;
     }
 
+    /**
+     * Get collection of Tables.
+     *
+     * @return ArrayObject<int, Table>
+     */
+    public function getTableCollection()
+    {
+        return $this->tableCollection;
+    }
+
+    /**
+     * Add Table.
+     *
+     * @return $this
+     */
+    public function addTable(Table $table): self
+    {
+        $table->setWorksheet($this);
+        $this->tableCollection[] = $table;
+
+        return $this;
+    }
+
+    /**
+     * Add Table Range by using numeric cell coordinates.
+     *
+     * @param int $columnIndex1 Numeric column coordinate of the first cell
+     * @param int $row1 Numeric row coordinate of the first cell
+     * @param int $columnIndex2 Numeric column coordinate of the second cell
+     * @param int $row2 Numeric row coordinate of the second cell
+     *
+     * @return $this
+     */
+    public function addTableByColumnAndRow($columnIndex1, $row1, $columnIndex2, $row2): self
+    {
+        $cellRange = Coordinate::stringFromColumnIndex($columnIndex1) . $row1 . ':' . Coordinate::stringFromColumnIndex($columnIndex2) . $row2;
+
+        return $this->addTable(new Table($cellRange, $this));
+    }
+
+    /**
+     * Remove collection of Tables.
+     */
+    public function removeTableCollection(): self
```

Comment:
Is it worth having options to remove individual tables (e.g. `removeTableByCellRange()` or `removeTableByName()`) or is it better to just get the table collection, remove the table that you want removing, and then `setTableCollection()` again?

Reply:
Definitely, its worthy. I prefer `removeTableByName()` in Worksheet and `removeTablefromWorksheet()` in Table.

Reply:
`removeTableByName()` added 50b91e8